<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="opsQueueTableView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>281</width>
      <height>251</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>10</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTableView" name="runningStatusTableView">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="memTableView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="reservationStationsTableView">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>741</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="opsQueueLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ops Queue</string>
    </property>
   </widget>
   <widget class="QLabel" name="runningStatusLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Running Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="loadQueueLabel">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Queue</string>
    </property>
   </widget>
   <widget class="QLabel" name="storeQueueLabel">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>190</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Store Queue</string>
    </property>
   </widget>
   <widget class="QLabel" name="memLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mem</string>
    </property>
   </widget>
   <widget class="QLabel" name="reservationStationsLabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>360</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Reservation Stations</string>
    </property>
   </widget>
   <widget class="QLabel" name="fuLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>640</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FU</string>
    </property>
   </widget>
   <widget class="QTableView" name="loadQueueTableView">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>371</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="fuTableView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>670</y>
      <width>1091</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="storeQueueTableView">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>210</y>
      <width>371</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addOpsButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oneCycleButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setOpButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nStepsButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-&gt;N</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addOpsFromFileButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cycles</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="CyclesBrowser">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="finishStepsButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-&gt;D</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1185</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>FILE</string>
    </property>
    <addaction name="actionAdd_Ops_From_File"/>
   </widget>
   <widget class="QMenu" name="menuOPS">
    <property name="title">
     <string>OPS</string>
    </property>
    <addaction name="actionAdd_Op"/>
    <addaction name="actionSet_Op"/>
    <addaction name="actionDelete_Op"/>
   </widget>
   <widget class="QMenu" name="menuSTEPS">
    <property name="title">
     <string>STEPS</string>
    </property>
    <addaction name="actionOne_Step"/>
    <addaction name="actionN_Steps"/>
    <addaction name="actionFinish_Steps"/>
   </widget>
   <widget class="QMenu" name="menuRESET">
    <property name="title">
     <string>RESET</string>
    </property>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuSET">
    <property name="title">
     <string>SET</string>
    </property>
    <addaction name="actionSet_Memory"/>
    <addaction name="actionSet_Register"/>
   </widget>
   <addaction name="menuFILE"/>
   <addaction name="menuOPS"/>
   <addaction name="menuSTEPS"/>
   <addaction name="menuRESET"/>
   <addaction name="menuSET"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_Ops_From_File">
   <property name="text">
    <string>Add Ops From File</string>
   </property>
  </action>
  <action name="actionAdd_Op">
   <property name="text">
    <string>Add Op</string>
   </property>
  </action>
  <action name="actionSet_Op">
   <property name="text">
    <string>Set Op</string>
   </property>
  </action>
  <action name="actionDelete_Op">
   <property name="text">
    <string>Delete Op</string>
   </property>
  </action>
  <action name="actionOne_Step">
   <property name="text">
    <string>One Step</string>
   </property>
  </action>
  <action name="actionN_Steps">
   <property name="text">
    <string>N Steps</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionSet_Memory">
   <property name="text">
    <string>Set Memory</string>
   </property>
  </action>
  <action name="actionSet_Register">
   <property name="text">
    <string>Set Register</string>
   </property>
  </action>
  <action name="actionFinish_Steps">
   <property name="text">
    <string>Finish Steps</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
